bundle/edu.gemini.catalog/src/main/scala/edu/gemini/catalog/api/CatalogQuery.scala

```diff
@@ -18,6 +18,21 @@ case class MagnitudeQueryFilter(mc: MagnitudeConstraints) extends QueryResultsFi
   def filter(t: SiderealTarget): Boolean = mc.filter(t)
 }
 
+sealed abstract class CatalogName(val id: String, val displayName: String)
+
+case object sdss extends CatalogName("sdss9", "SDSS9 @ Gemini")
+case object gsc234 extends CatalogName("gsc234", "GSC234 @ Gemini")
+case object ppmxl extends CatalogName("ppmxl", "PPMXL @ Gemini")
+case object ucac4 extends CatalogName("ucac4", "UCAC4 @ Gemini")
+case object twomass_psc extends CatalogName("twomass_psc", "TwoMass PSC @ Gemini")
+case object twomass_xsc extends CatalogName("twomass_xsc", "TwoMass XSC @ Gemini")
+case object simbad extends CatalogName("simbad", "Simbad")
+
```

Comment:
It's important to capitalize the names of case objects because otherwise you cannot pattern-match on them without some trickery. For example, this method will always return `0` because lowercase identifiers in patterns match everything.

```scala
def foo(cn: CatalogName): Int = 
  cn match {
    case sdss        => 0
    case gsc234      => 1
    case ppmxl       => 2
    case ucac4       => 3
    case twomass_psc => 4
    case twomass_xsc => 5
    case simbad      => 6
  }
```

So unless you have an *extremely* good reason for keeping these lower-case, I think you should switch them,